File: app/src/main/java/com/calculatemypace/david/calculateMyPace/FirstActivity.java
package com.calculatemypace.david.calculateMyPace;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageButton;

public class FirstActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_first);

        Button paceButton = (Button) findViewById(R.id.pace);

        paceButton.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                startActivity(new Intent(FirstActivity.this, MainActivity.class));
            }
        });

        Button distanceButton = (Button) findViewById(R.id.stopwatch);

        distanceButton.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                startActivity(new Intent(FirstActivity.this, StopwatchActivity.class));
            }
        });

        ImageButton button = (ImageButton) findViewById(R.id.imageButton);

        button.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                startActivity(new Intent(FirstActivity.this, StopwatchActivity.class));
            }
        });
    }

    
}

File: app/src/main/java/com/calculatemypace/david/calculatemypace/FirstActivity.kt
package com.calculatemypace.david.calculatemypace

import android.app.Activity
import android.os.Bundle
import android.content.Intent
import android.util.Log
import android.view.View
import android.widget.Button
import android.widget.ImageButton
import butterknife.BindView
import butterknife.ButterKnife
import butterknife.OnClick

class FirstActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_first)
        ButterKnife.bind(this)

    }

    @OnClick(R.id.pace_button)
    fun startMainActivity() {
        startActivity(Intent(this, MainActivity::class.java))
    }

    @OnClick(R.id.stopwatch)
    fun startStopwatchActivity() {
        startActivity(Intent(this, StopwatchActivity::class.java))
    }

    @OnClick(R.id.imageButton)
    fun startSecretActivity() {
        startActivity(Intent(this, StopwatchActivity::class.java))
    }

}